modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java

```diff
@@ -140,6 +142,11 @@ public String getIdName() {
 		return _idName;
 	}
 
+	@Override
+	public String getNamespace() {
+		return "Object_" + StringUtil.upperCaseFirstLetter(_companyName);
```

Comment:
A field starting with a number can't be a field in GraphQL right? this is very inconsistent compared with REST.

Reply:
That's it... My first approach was using the same strategy as REST with only the companyId, but the field must not start with a number

Reply:
I've finally use a camel case version of the company name because using encodeURL creates characters that were not allowed in a GraphQL field (+ or %). This way we maintain consistency between REST and GraphQL

Reply:
Oh I see... why camel case? do we need camel case? By why do you concatenate with Object_?